.github/workflows/publish-rust.yml

```
@@ -102,6 +102,32 @@ jobs:
       - name: Run clippy
         run: ./scripts/check-clippy.sh
 
+  docsrs:
+    name: Docs.rs
+    needs: [sanity]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v4
+
+      - name: Install 'nightly' toolchain
+        uses: dtolnay/rust-toolchain@nightly
+
+      - name: Run docs.rs container build
+        run: |
+          docker run --rm \
+            -v ~/.cargo:/cargo \
+            -v ~/.rustup:/rustup \
+            -v "$PWD":/work \
+            -w /work \
+            -e CARGO_HOME=/cargo \
+            -e RUSTUP_HOME=/rustup \
+            -e PATH=/cargo/bin:$PATH \
+            -e DOCS_RS=1 \
+            -e RUSTDOCFLAGS="--cfg docsrs" \
```

I'm not sure if you ran this command successfully on your machine, but on mine, I need to quote all of the env declarations
```suggestion
            -e "CARGO_HOME=/cargo" \
            -e "RUSTUP_HOME=/rustup" \
            -e "PATH=/cargo/bin:$PATH" \
            -e "DOCS_RS=1" \
            -e "RUSTDOCFLAGS=\"--cfg docsrs\"" \
```